guides/guides.js

```diff
@@ -0,0 +1,156 @@
+#!/usr/bin/env node
+
+/*
+ * Copyright 2024 Red Hat Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+const https = require('follow-redirects').https
+const path = require('node:path')
+const fs = require('fs')
```

Comment:
Let's be consistent and use the `node:` prefix for all built-in Node.js modules.

```js
import fs from 'node:fs'
```

Reply:
Done! All import now.